easybuild/easyconfigs/g/genometools/genometools-1.5.8-foss-2015b-no-cairo.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'MakeCp'
+
+name = 'genometools'
+version = '1.5.8'
+versionsuffix = '-no-cairo'
```

Comment:
There's little point in having easyconfigs for `genometools` both with and without cairo, since the former would provide everything the latter does.